### PR TITLE
Build improvements

### PR DESCRIPTION
**Pull request by evansiroky**
- Remove unneeded packages
- Get react-hot-loader working
- Get css-hot-loader working
- disable redux-logger during production builds

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -64,7 +70,7 @@ const render = App => ReactDOM.render(
   document.getElementById('main')
 )
 
-render(TrimetWebapp)
+hot(render(TrimetWebapp))
```
Comment (landonreed) [inline lib/main.js]: This probably deserves a comment. It would appear that hot loading would be enabled at all times based on how this is called.

Comment (landonreed) [reply lib/main.js]: Ah, I see now that this is to keep the state in place during a hot reload. Still deserves a comment and description of dev vs. prod.

**Comment by landonreed** [comment]
Using this branch with `watch-build.sh` and changing `fa-map-marker` to `fa-bath` in `location.icon.js` gave me this error:
image

**Comment by landonreed** [comment]
I've confirmed that trying to do the same with `master` works fine (the page reloads, but at least there is no error).

**Comment by evansiroky** [comment]
Ugh, I think you found the one thing that doesn't work with hot-reloading. Font-awesome classes aren't updating well when within a react-leaflet context. A manual refresh may be needed in this case.

**Comment by evansiroky** [comment]
@landonreed, lmk if you think font-awesome icons not hot-reloading is a blocking issue.

**Comment by landonreed** [comment]
I suppose it doesn't need to be blocking -- but it does make me wonder what else might be broken with hot reloading... maybe I just did really get lucky and choose the only thing that will cause issues 🤷‍♂ .

**Pull request merged**